src/networks.ts

```diff
@@ -33,6 +34,8 @@ export function getRpc(network: SupportedNetworks): string {
       return "https://mainnet.optimism.io"
     case "Avalanche":
       return "https://api.avax.network/ext/bc/C/rpc"
+    case "Lens":
+        return "https://rpc.lens.xyz";
```

Comment:
Nit for consistent spacing:

```suggestion
      return "https://api.avax.network/ext/bc/C/rpc"
    case "Lens":
      return "https://rpc.lens.xyz";
```